src/config/schema.rs

```diff
@@ -54,6 +54,9 @@ pub enum ObjectStore {
     InMemory(InMemory),
     #[cfg(feature = "object-store-s3")]
     S3(S3),
+    #[cfg(feature = "object-store-gcs")]
+    #[serde(rename = "gcs")]
```

Comment:
I'm not sure why, but if I don't add this, we get this
![image](https://user-images.githubusercontent.com/182515/235083243-52276214-38e2-4b61-956a-8a8554afbdac.png)

Reply:
I think it's just how serde automatically names it and it looks more like an abbreviation to it, so it uses that format. What you did is fine.

Reply:
TIL serde has "abbreviation awareness." Cheers! 

Marking resolved.